docker/2.0.0/py3/Dockerfile.cpu

```diff
@@ -17,101 +32,91 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
+ && cp /root/.ssh/id_rsa.pub /root/.ssh/authorized_keys \
+ && printf "Host *\n  StrictHostKeyChecking no\n" >> /root/.ssh/config
 
 WORKDIR /
 
-ARG PYTHON=python3
-ARG PYTHON_PIP=python3-pip
-ARG PIP=pip3
-ARG PYTHON_VERSION=3.6.6
-
-RUN wget https://www.python.org/ftp/python/$PYTHON_VERSION/Python-$PYTHON_VERSION.tgz && \
-    tar -xvf Python-$PYTHON_VERSION.tgz && cd Python-$PYTHON_VERSION && \
-    ./configure && make && make install && \
-    apt-get update && apt-get install -y --no-install-recommends libreadline-gplv2-dev libncursesw5-dev libssl-dev libsqlite3-dev tk-dev libgdbm-dev libc6-dev libbz2-dev && \
-    make && make install && rm -rf ../Python-$PYTHON_VERSION* && \
-    ln -s /usr/local/bin/pip3 /usr/bin/pip
-
-ENV PYTHONDONTWRITEBYTECODE=1 PYTHONUNBUFFERED=1 PYTHONIOENCODING=UTF-8 LANG=C.UTF-8 LC_ALL=C.UTF-8
-
-ARG framework_support_installable=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
-ARG sagemaker_tensorflow_extensions=tensorflow-2.0.0-cp36-cp36m-manylinux2010_x86_64.whl
-COPY $framework_support_installable .
-COPY $sagemaker_tensorflow_extensions .
+###########################################################################
+# Python won’t try to write .pyc or .pyo files on the import of source modules
+ENV PYTHONDONTWRITEBYTECODE=1 
+ENV PYTHONUNBUFFERED=1 
+ENV PYTHONIOENCODING=UTF-8 
+ENV LANG=C.UTF-8 
+ENV LC_ALL=C.UTF-8
 
+# Use THENSORFLOW_WHL instead of TF_URL before releasing
 # ARG TF_URL="https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.14/AmazonLinux/cpu/final/tensorflow-1.14.0-cp36-cp36m-linux_x86_64.whl"
 
-RUN ${PIP} --no-cache-dir install --upgrade pip setuptools
+ARG FRAMEWORK_SUPPORT_INSTALLABLE=sagemaker_tensorflow_container-2.0.8.dev0.tar.gz
+ARG THENSORFLOW_WHL=tensorflow-2.0.0-cp36-cp36m-manylinux2010_x86_64.whl
+COPY $FRAMEWORK_SUPPORT_INSTALLABLE .
+COPY $THENSORFLOW_WHL .
+
+RUN ${PIP} --no-cache-dir install --upgrade \
+    pip \
+    setuptools
 
 # Some TF tools expect a "python" binary
-RUN ln -s $(which ${PYTHON}) /usr/local/bin/python
+RUN ln -s $(which ${PYTHON}) /usr/local/bin/python \
+ && ln -s $(which ${PIP}) /usr/bin/pip
 
 RUN ${PIP} install --no-cache-dir -U \
-           numpy==1.16.4 \
-           scipy==1.2.2 \
-           scikit-learn==0.20.3 \
-           pandas==0.24.2 \
-           Pillow==6.1.0 \
-           h5py==2.9.0 \
-           keras_applications==1.0.8 \
-           keras_preprocessing==1.1.0 \
-           keras==2.2.4 \
-           requests==2.22.0 \
-           awscli==1.16.196 \
-           mpi4py==3.0.2 \
-           $sagemaker_tensorflow_extensions && \
-    #       "sagemaker-tensorflow>=1.14,<1.15" && \
+    numpy==1.17.4 \
+    scipy==1.2.2 \
+    scikit-learn==0.20.3 \
+    pandas==0.24.2 \
+    Pillow==6.1.0 \
+    h5py==2.9.0 \
+    keras_applications==1.0.8 \
+    keras_preprocessing==1.1.0 \
+    keras==2.3.1 \
+    requests==2.22.0 \
+    awscli==1.16.196 \
+    mpi4py==3.0.2 \
+    # "sagemaker-tensorflow>=1.14,<1.15" && \
```

Comment:
I believe they released the sagemaker-tensorflow==2.0.0.1.0.0

Reply:
yup

Reply:
added